pycromanager/test/test_viewer.py

```diff
@@ -0,0 +1,45 @@
+import os
+import pytest
+from pycromanager import Acquisition, Core, multi_d_acquisition_events
+import napari
+
+# Skip tests in this module if it is running in GitHub Actions, which does not support NDViewer
+# https://docs.pytest.org/en/7.1.x/how-to/skipping.html
+# https://docs.github.com/ko/actions/learn-github-actions/variables
+pytestmark = pytest.mark.skipif(os.getenv('GITHUB_ACTIONS') == 'true', reason="NDViewer does not work in GitHub Actions")
```

Comment:
Currently, the viewer is the most fragile part of `pycromanager` (e.g. #530) as the viewer tests do not run in GitHub actions and need to be run manually.

I'm not sure if these viewer tests could run in GitHub actions, I'll try. It would be good if we could tests as much of the viewer code as possible in an automated manner.

Reply:
Yes, agreed. Although Im not sure if its possible. You'd need to have a machine with a graphics environment and Im not sure if the virtual machines of  GH actions can do that